### ISSUE TITLE
Not working on Dying Light 2, no pop-up window.

### ISSUE THREAD

**Issue by iNORIcc** [Nukem9/dlssg-to-fsr3#96]
Dying Light 2 is not working properly. It does not open a pop-up window when I run the game. It also does not have the FG option. This is working fine for Cyberpunk 2077, Alan Wake 2, and The Witcher 3.

**Comment by Daineli** [comment]
Yep, I tested normal version and tweaks version, neither not respond.

**Comment by NightGuardianX** [comment]
I've got a pop-up window via DLSS Tweaks, but see no option in game anyway

**Comment by Daineli** [comment]
The same, ver 0.5 dlss tweaks, pop-up window but in game is still missing option.
